docs/modules/ROOT/pages/runbooks/APPUiOCloudReportingMissingData.adoc

```diff
@@ -36,7 +36,9 @@ If the pod has been deleted, you can also use https://logging.apps.cloudscale-lp
 Use the content of the field `source` as a starter.
 Use https://portal.appuio.cloud to look up the organization, and then use https://control.vshn.net to identify a matching customer.
 Once a matching customer was found, look it up in https://erp.vshn.net.
-Copy the ID from the URL.
+Search for the customer's accounting contact.
```

Comment:
How is a customer's accounting contact marked in Odoo?

Reply:
It is named `<customer name>, Accounting`.